ynput/ayon-core
# Chore: Move substance painter client code

## Changelog Description
This PR is to move substance painter integration into the client code for the sake of splitting repos for each host in the future.

## Additional info

## Testing notes:
1. Build substance painter addon and core addon with this branch
2. Update substance painter addon and core addon
3. n/a
4. Launch AYON
5. Should be working as expected.
